src/behaviors/JSON.js

```diff
@@ -83,6 +83,14 @@ var JSON = Local.extend('behaviors.JSON', {
     getHeaders: function() {
         return this.dataModel.getHeaders();
     },
+    /**
+     * @memberOf behaviors.JSON.prototype
+     * @return {string} The field at `colIndex`.
+     * @param {number} colIndex - the column index of interest
+     */
+    getField: function(colIndex) {
+        return colIndex === -1 ? 'tree' : this.getColumnFromFullList(colIndex).getField();
```

Comment:
This has performance implications as anyone calling this will not only be searching the visible list of columns